frontend/src/store/index.js

```diff
@@ -749,8 +763,8 @@ export const store = new Vuex.Store({
           context.commit('setUser', resp.data.csr)
           let officeType = resp.data.csr.office.sb.sb_type
           context.commit('setOffice', officeType)
-          context.commit('setDefaultCounter', resp.data.csr.office.counters.filter(
-            c => c.counter_name === DEFAULT_COUNTER_NAME)[0])
+         // context.commit('setDefaultCounter', resp.data.csr.office.counters.filter(
```

Comment:
Are these the things that are broken from the other changes that were merged? Have you tested them with the latest merge into `sprint-10-4`? Since all the manage.py bootstrap stuff now works, I would hope that this does now too.